File: leetcode2/1easy/이준열/3178.cpp
class Solution {
    public:
        int numberOfChild(int n, int k) {
            int dir = 1;
            int cur  = 0;
            
            for (int i = 0; i < k; i++)
            {
                if ((i != 0)
                    && ((cur == 0) || (cur == n-1)))
                    dir *= -1;
                cur += dir;
            }
    
            return cur;
        }
    };

File: leetcode2/1easy/유시연/3178.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def numberOfChild(self, n: int, k: int) -> int:
        i = -1
        go_right = True

        while k >= 0:
            if i == 0:
                go_right = True
            if i == n-1:
                go_right = False

            if go_right:
                i += 1
                k -= 1

            else:
                i -= 1
                k -= 1
                
        return i

        